.github/workflows/end2end.yaml

```diff
@@ -29,3 +35,32 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
           java-package: jdk
+
+      - name: Cache Maven packages
+        uses: actions/cache@v2.1.6
```

Comment:
Just FYI, this action can be removed by adding `cache: 'maven'` in the `setup-java` action. They are equivalent.

Reply:
thanks a lot for the into: do you mind open a PR with such a change please?